### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by alicia-alcaide**

### CODE REVIEW

File: staff/alicia-alcaide/precourse/bingo/index.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    };
+    return number;
+};
+
+
+function matchNumInCard(number, bingoCard) {
+    var posicion = findNumInCard(number, bingoCard);
+    if (posicion.row >= 0 && posicion.col >= 0) {
+        bingoCard[posicion.row][posicion.col].matched = true;
+        return true
```
Comment (bilalelbedoui93) [inline staff/alicia-alcaide/precourse/bingo/index.js]: the semicolon is missing

File: staff/alicia-alcaide/precourse/bingo/index.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    } else {
+        return false
```
Comment (bilalelbedoui93) [inline staff/alicia-alcaide/precourse/bingo/index.js]: the semicolon is missing

File: staff/alicia-alcaide/precourse/bingo/index.js (hunk 1 above)
Comment (alicia-alcaide) [reply staff/alicia-alcaide/precourse/bingo/index.js]: done

File: staff/alicia-alcaide/precourse/bingo/index.js (hunk 2 above)
Comment (alicia-alcaide) [reply staff/alicia-alcaide/precourse/bingo/index.js]: done

**Pull request merged**